addons/mail/static/src/components/call_sidebar_view/call_sidebar_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="mail.CallSidebarView" owl="1">
+    <t t-if="callSidebarView">
+        <div class="o_CallSidebarView d-flex align-items-center h-100 flex-column">
+            <t t-foreach="callSidebarView.callView.tileParticipantCards" t-as="participantCard" t-key="'grid_tile_'+participantCard.localId">
+                <t t-if="!callSidebarView.callView.filterVideoGrid or (participantCard.rtcSession and participantCard.rtcSession.videoStream)">
+                    <!-- maybe filter focused partner out? -->
+                    <CallParticipantCard
+                        className="'o_CallSidebarView_participantCard w-100'"
+                        record="participantCard"
+                    />
+                </t>
+            </t>
```

Comment:
Not something to change in this PR, as to not increase its scope.
But a remark for following changes:

The model `CallParticipantCard` shares context between card in sidebar and card in tile.
This is a problem, because it makes it hard to put context-specific code.
Also the `t-if` below `t-foreach` filter some cards from the tile, which shows that there is some contextual info that are manually handled here. It's kinda working, but scales poorly.

The solution would be to have `CallParticipantCard` as a logic model, and introduce `CallParticipantCardView`.
Have 2 distincts `many` fields on `ParticipantCardView`, one being the tiles and the other being sidebar.
The component `CallParticipantCard` should use the `CallParticipantCardView` as record, instead of `CallParticipantCard`.

Doing so would put control on views of which card in the compute method of `sidebarCardViews` field.
And also would allow us to define view-specific behaviour, in case we need/want it.